### PR TITLE
updates

### PR DESCRIPTION
**Pull request by steveoh**
this pr 

- replaces the 👎🏼 with ⛔ 
- allows the tool to be run locally

### CODE REVIEW

File: src/conductor/checks.py [hunk 1]
```diff
@@ -306,10 +306,10 @@ class GSheetChecker():
     client = None
     worksheet = None
 
-    def __init__(self, table, sheet_id, worksheet_name, testing=False):
+    def __init__(self, table, sheet_id, worksheet_name, sa):
```
Comment (stdavis) [inline src/conductor/checks.py:309]: **nitpick**: More descriptive variable name? I had to read a few lines to figure out that `sa` is likely short for "service account".

File: tests/test_checks.py [hunk 2]
```diff
@@ -401,15 +401,18 @@ def test_sheets_returns_true_if_neighbors_all_have_values(mocker):
 
 @pytest.mark.google
 def test_sheets_can_find_workspace():
-    patient = GSheetChecker('fake.table', '11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ', 'SGID Stewardship Info')
+    patient = GSheetChecker(
+        'fake.table', '11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ', 'SGID Stewardship Info', 'client-secret.json'
```
Comment (stdavis) [inline tests/test_checks.py:405]: **question**: Does this depend on there being a `client-secret.json` file in the project? Or is that taken care of in the testing framework?

Comment (steveoh) [reply tests/test_checks.py:405]: all the tests marked as `google` require a service account json file. It's doc'd in the readme under local development.

File: src/conductor/checks.py (hunk 1 above)
Comment (steveoh) [reply src/conductor/checks.py:309]: ok i can make that more descriptive.

**Comment by stdavis** [comment]
This is awesome! Being able to run locally is 💯

**Comment by steveoh** [comment]
Ok, i'll build a new image and push it to the cloud. we are teetering on the cost vs usefulness of cloud build and a CI/CD pipeline if we keep pushing more features.

**Comment by stdavis** [comment]
Maybe just build a deploy shell script and include it in the project?

**Pull request merged**